client/src/components/DateSelector/DateSelector.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import styles from "./DateSelector.module.css";
+import { PropTypes } from "prop-types";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+export const DateSelector = ({ onDateChanged, showTime }) => {
+  const [currDate, setCurrDate] = useState(new Date());
+
+  const _handleDateChanged = (newDate) => {
```

Comment:
This change handler doesn't need an underscore on the handle

Reply:
Changed as requested.